src/components/input/input.md

```diff
@@ -246,3 +246,14 @@ Type: `textarea`
         placeholder="write a story..."
       />
     </div>
+
+Type: `editable`
+
+    <div>
+      <span>What is your favorite color?</span>
+      <Input
+        label="My favorite color is"
+        value="Blue"
```

Comment:
Did you just assume my favourite color??

Reply:
Isn't blue everyone's favourite? 🤔